<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DoktorWindow</class>
 <widget class="QMainWindow" name="DoktorWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>793</width>
    <height>596</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Ambulanta</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QComboBox" name="pacijentCombo">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>290</y>
      <width>241</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="currentText">
     <string extracomment="Pacijent"/>
    </property>
   </widget>
   <widget class="QLabel" name="pacijentIzaberi">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>250</y>
      <width>231</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Izaberite doktora:</string>
    </property>
   </widget>
   <widget class="QLabel" name="pacijentZakazi_2">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>10</y>
      <width>231</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Zakažite termin</string>
    </property>
   </widget>
   <widget class="QCalendarWidget" name="calendarWidget">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>60</y>
      <width>280</width>
      <height>171</height>
     </rect>
    </property>
   </widget>
   <widget class="QGraphicsView" name="pacijentGrafik">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>190</y>
      <width>381</width>
      <height>361</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pacijentZakazi">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>350</y>
      <width>211</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Zakaži termin</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pacijentPrikazi">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>191</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Prikaži broj</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pacijentObrisi">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>150</y>
      <width>191</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Obriši broj</string>
    </property>
   </widget>
   <widget class="QLabel" name="pacijentGrafikLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>411</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>17</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Broj pacijenata kod svih doktora:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pacijent_pocetni">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>510</y>
      <width>201</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Početni ekran</string>
    </property>
   </widget>
   <widget class="QLabel" name="ime_pacijenta">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>311</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>17</pointsize>
     </font>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
